doc/api/errors.md

```diff
@@ -1502,6 +1502,12 @@ a hostname in the first parameter.
 An excessive amount of TLS renegotiations is detected, which is a potential
 vector for denial-of-service attacks.
 
+<a id="ERR_TLS_RENEGOTIATION_DISABLED"></a>
+### ERR_TLS_RENEGOTIATION_DISABLED
+
+This errors is triggered when attempts are made to renegotiate TLS on a socket 
+instance which has TLS  disabled 
```

Comment:
The description is not quite accurate. Perhaps:

```
Triggered when an attempt is made to renegotiate the secure configuration on a TLS
socket after renegotiation has been disabled.
```

Reply:
For consistency with other error descriptions, this should start with `An attempt was made...`